internals/cli/cmd_enter.go

```diff
@@ -178,7 +179,7 @@ func (cmd *cmdEnter) Execute(args []string) error {
 	case runStop = <-runReadyCh:
 	case runPanic := <-runResultCh:
 		if runPanic == nil {
-			panic("internal error: pebble daemon stopped early")
+			panic(fmt.Sprintf("internal error: %s daemon stopped early", cmdpkg.ProgramName))
```

Comment:
IMO we should just drop this one and just say "daemon" or "service manager" (depending on what we do elsewhere).